packages/kbn-eslint-config/.eslintrc.js

```diff
@@ -297,5 +298,6 @@ module.exports = {
     '@kbn/imports/uniform_imports': 'error',
     '@kbn/imports/no_unused_imports': 'error',
     '@kbn/imports/no_boundary_crossing': 'error',
+    '@kbn/telemetry/event_generating_elements_should_be_instrumented': 'warn',
```

Comment:
Hey @mistic! I don't believe I follow your question.

In CI as you say there won't be any noise as warnings as suppressed.

In the IDE there will be noise on the elements targeted, but that is by design as the rule is meant to nudge engineers to add the prop. Are you worried that noise will be too large?

Reply:
I don't believe it is wise to set this rule to error at this stage for the reasons described in the PR description: there may be a legitimate reason why an engineer might decide to not add this prop to a targeted element. I don't think we should block the engineer from merging in this scenario.

Reply:
I guess what I was trying to communicate is that we had long accumulated knowledge on Kibana that warnings quickly add up to something that no one looks into or cares enough about to fix and we came to the conclusion that in that case warnings are not useful. Or we do care enough about something to make it an error or we don't and just don't do anything. For that case I can understand your logic around the warning thing in terms of a development suggestion but I was just exposing our internal view in the operations team to understand if it would make sense to make it an error for this case. Sure it will block people at first on those suggestions but an `eslint-ignore` comment on those situations people just don't want to do anything will work out.I'm not sure if in that case this setup it will make sense due to the number of potential suggestions but I would left that for your to decide (having in account the potential IDE noise too) 😃

Reply:
I can get behind that. I was worried the DX might go down if the rule is set to error, leading to unhappy engineers (which is definitely not my intention) but I trust you have more experience in how these types of changes are received by engineers. I'll update the PR to set it to error. Thanks for brainstorming this with me 😃